templates/content_based_routing/src/content_based_routing/main.bal

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/http;
+import ballerina/log;
+import ballerina/config;
+
+string ADD_OPERATION = "add";
+string SUBTRACT_OPERATION = "subtract";
+
+listener http:Listener clientListener = new http:Listener(config:getAsInt("LISTENER_PORT"));
+http:Client arithmeticServiceEP = new(config:getAsString("CLIENT_ENDPOINT"));
+
+@http:ServiceConfig {
+    basePath: "/calculatorService"
+}
+service calculatorService on clientListener {
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/calculate"
+    }
+    resource function calculate(http:Caller caller, http:Request request) {
+
+        http:Response|error response;
+        http:Response errorResponse = new;
+
+        var requestPayload = request.getJsonPayload();
+
+        if(requestPayload is json) {
+            string operation = <string>requestPayload.operation;
+
+            if(operation == ADD_OPERATION) {
```

Comment:
It would be efficient if we use the match operator

Reply:
Fixed.